src/hardware/keyboard.cpp

```diff
@@ -434,9 +434,13 @@ void KEYBOARD_AddKey(KBD_KEYS keytype,bool pressed) {
 	case KBD_pagedown:extend=true;ret=81;break;
 	case KBD_insert:extend=true;ret=82;break;
 	case KBD_delete:extend=true;ret=83;break;
-	case KBD_leftgui:extend=true;ret=90;break;
-	case KBD_rightgui:extend=true;ret=89;break;
-	
+
+	 // The IBM PC XT and AT keyboard layouts lack GUI keys, so we don't
+	 // generate event codes for them. For reference, see the IBM Model F XT
```

Comment:
I think removing the scan codes from keyboard.cpp is not a good idea - one day we might want to officially support Windows 95 and beyond, it would be nice to have the scan codes available then.

If our GFX subsystem don’t send problematic keys to the keyboard emulation, it’s enough.

Reply:
Even on Linux with a PC standard keyboard, my right-GUI key generates `/` in DOS.

Maybe this is better: `if (DOSVER < 7)` then we know DOS is pre-Windows 95 and therefore DOS doesn't understand GUI keys (so just selectively ignore)?

Reply:
But do we have a way to tell which version of MS-DOS is installed? What about non-Microsoft DOS systems, like DR-DOS, PC-DOS, and so on?

Reply:
Right; we can only use it for the internal/emulated keysteoke handling.

 If the user `boot`s their own DOS kernel, then I think it's up that kernel what it does with the scan codes.

Reply:
AFAIK DOS’es rely on the BIOS to interpret scancodes for them. It seems the current implementation in https://github.com/dosbox-staging/dosbox-staging/blob/main/src/ints/bios_keyboard.cpp cannot handle Win keys - but how it behaves upon receiving an unknown scancode, I don’t know.

Reply:
confirmed that's the case, @FeralChild64 (I added logging into `keyboard.cpp switch statement, and indeed, it's still active even when booting).